src/views/coordinator/coordinator.view.jsx

```diff
@@ -180,7 +180,6 @@ function Coordinator ({
                     >
                       <BidsList
```

Comment:
I think that we also need to remove the `totalNumberOfBidsInSlot` prop from the `BidsList` component right?

Reply:
Nope, because it is being used in slot view.

Reply:
Ahh okay! And we don't need to perform any kind of check when we are not passing down this prop in the slot view?

Reply:
I don't think so.